testdata/authorization/ldapserver.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
One minor issue, it is irrelevant to authorization, thus testdata/authorization seems not proper? How about testdata/pods/ ?

Reply:
Done. Thanks